src/commands/clean.rs

```diff
@@ -20,9 +23,17 @@ pub async fn clean(_: Clean, global: GlobalArgs) -> crate::error::Result<()> {
             );
         }
     }
-    let venv_dir = project_venv_dir(&global.project);
+    let venv_dir = project_venv_dir(path);
     if venv_dir.exists() {
-        if let Err(err) = tokio::fs::remove_dir_all(&venv_dir).await {
+        if venv_dir.is_symlink() {
+            if let Err(err) = tokio::fs::remove_file(&venv_dir).await {
+                tracing::warn!(
+                    "Failed to remove project venv symlink at {}: {}",
+                    venv_dir.display(),
+                    err
+                );
+            }
+        } else if let Err(err) = tokio::fs::remove_dir_all(&venv_dir).await {
```

Comment:
Would this fail in the unlikely event `venv_dir` is a file?

Reply:
Yeah, but the error message is clear enough and its not a regression if it does. I wouldn't want to account for weird user behavior. If they've placed that file there for some reason, I also wouldn't want to delete it